src/modules/like.js

```diff
@@ -0,0 +1,27 @@
+import { getLikes, postLike } from './api.js';
+
+// Post like function
+export const like = async () => {
+  const likeBtn = document.querySelectorAll('.like-btn');
+  likeBtn.forEach((icon) => {
+    icon.addEventListener('click', async () => {
+      const { id } = icon;
+      postLike(id);
+    });
+  });
```

Comment:
Kindly add counter function inside this function because when users click the like button, it will need to count.
You can only add below code in your function.

` let count = +icon.nextElementSibling.innerHTML;`
 ` count++;`
  `icon.nextElementSibling.innerHTML = count;`

Reply:
Sure I will make these changes. Thank you 🤝